app/Models/Training/WaitingList.php

```diff
@@ -175,6 +179,28 @@ public function getFormattedDepartmentAttribute()
         };
     }
 
+    public function getShouldCheckAtcHoursAttribute(): bool
+    {
+        // default to true
+        return $this->isAtcList() && ($this->feature_toggles['check_atc_hours'] ?? true);
```

Comment:
Why is this not just looking at the toggle? Would be much cleaner and obvious to the user as to what is going on. Also would remove the need for the comment.

Reply:
Historically we have only shown cts theory exams & atc hour checks on ATC lists so this is just maintaining backwards compatibility.

Reply:
changed